toolbox/ci2gubernator/README.md

```diff
@@ -62,7 +66,7 @@ At the end of a build, execute the following command to
 * upload `finished.json`, `artifacts/junit_*.xml`, and `build-log.txt` to GCS
 * update `latest-build.txt` of this job with the current build number
 
-Caveat: this command exits with the same exit code as supplied by --exit_code.
+The same rules on `--presubmit` usage apply in here.
```

Comment:
this is out of date.. In fact I am not sure if we need this sentence at all.

Reply:
Good catch. Fixed it in latest commit, but I would prefer to keep this line just to remind the users to not miss it.